cardano-api/src/Cardano/Api/Fees.hs

```diff
@@ -915,6 +915,7 @@ data BalancedTxBody era
 makeTransactionBodyAutoBalance
   :: forall era mode.
      IsShelleyBasedEra era
+  => ShelleyEraTxBody (ShelleyLedgerEra era)
```

Comment:
Likewise here, we don't want to avoid adding constraints at the top level

Reply:
Removed.